source/qml/Samples/QmlVisualizer/main.cpp

```diff
@@ -24,7 +24,7 @@ int main(int argc, char* argv[])
     SampleCardModel model;
     model.setList(&cardList);
 
-    const QString qmlString = model.generateQml(QString::fromStdString(Samples::card_Empty));
+    const QString qmlString = model.generateQml(QString::fromStdString(Samples::card_CheckboxInput));
```

Comment:
Why are we changing this again and again!!!

Reply:
Apologies, won't happen again